README.md

```diff
@@ -27,11 +27,22 @@ func main() {
 		"gedditAgent v1",
 	)
 
+	// Set listing options
+	subOpts := geddit.ListingOptions{
+		Limit: 10,
+	}
+
+	// Used to specify default subreddit sort
+	defaultPop := geddit.DefaultPopularity
```

Comment:
idk if this block of code is necessary, let's just inline `reddit.DefaultPopularity`.